modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@jimschubert Sorry, just recognized I answered not all of your questions. The issue with `**/*.cs` was, that `*.cs` files from the `obj/` dir were included, causing in some duplication compile errors.

We are using swagger-codegen in a Xamarin.Forms project. If you need further information, just mail me.